### ISSUE TITLE
How can I get the pretrain checkpoint ? 

### ISSUE THREAD

**Issue by burin-n** [FactoDeepLearning/LinePytorchOCR#1]
After I tried running `git lfs pull`, I found these following errors:

```
batch response: Git LFS is disabled for this repository.
error: failed to fetch some objects from 'https://github.com/FactoDeepLearning/LinePytorchOCR.git/info/lfs'
```

**Comment by FactoDeepLearning** [comment]
It seems to be a quota problem...

It should be okay there: https://git.litislab.fr/dcoquenet/linepytorchocr

**Comment by burin-n** [comment]
Thanks

**Comment by burin-n** [comment]
Are you sure that the checkpoint is the best one?

**Comment by FactoDeepLearning** [comment]
Yes, it's the one leading to 7.99% of CER for the IAM test set.

**Comment by burin-n** [comment]
I found the performance mismatch using the provided checkpoint.
Here is my result on IAM validation set.
```
cer: 0.0718
wer: 0.3554
nb_words: 18.2377
used samples : 42074/42074
```
And this is my result on IAM test set.
```
cer: 0.0979
wer: 0.4057
nb_words: 17.7860
used samples : 125616/125616
```

How did you process transcripts in `lines.txt`? 
I just replaced `|` with space.
Is there any preprocessing I missed?

**Comment by burin-n** [comment]
I see. There are differences between transcription in `lines.txt` from `ascii.tgz` and transcription in `xml.tgz`.

**Comment by FactoDeepLearning** [comment]
This should be it. I have release code to pre-format the IAM dataset to be sure.

Also, I advise you to check https://github.com/FactoDeepLearning/VerticalAttentionOCR.
Full code and weights are also available and we reached 4.95% for the IAM test set on lines.

**Comment by burin-n** [comment]
Got it. Thanks
